_opensearch/rest-api/ingest-apis/create-update-ingest.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Create or update ingest pipeline
+parent: Ingest APIs
+grand_parent: REST API reference
+nav_order: 11
+---
+
+# Create and update a pipeline
+
+The create ingest pipeline API operation creates or updates an ingest pipeline. Each pipeline requires an ingest definition defining how each processor transforms your documents. 
+
+## Example
+
+```
+PUT _ingest/pipeline/{id}
+{
+  "description" : "A description for your pipeline",
+  "processors" : [
+    {
+      "set" : {
+        "field": "field-name",
+        "value": "value"
+      }
+    }
+  ]
+}
+```
+
+## Path and HTTP methods
+```
+PUT _ingest/pipeline/{id}
+```
+
+## Request body fields
+
+Field | Type | Description
+:--- | :--- | :---
+`description` (optional) | string | Description of your ingest pipeline. 
+`processors` | processor objects | A processor that transforms documents. Runs in the order specified. Appears in index once ran.
```

Comment:
Just to be consistent with the rest of the site, unless all the parameters/fields are optional, I would add a 4th column to these tables to specify whether these parameters or fields are required.

Also, should "processor objects" be "Array of processor objects"?

And capitalize "string" and "processor" in type columns.

Reply:
Also, you don't have to monospace these fields